annotation/app/models.py

```diff
@@ -91,6 +91,11 @@ class AnnotatedDoc(Base):
     validated = Column(ARRAY(INTEGER), nullable=False, server_default="{}")
     tenant = Column(VARCHAR, nullable=False)
     task_id = Column(INTEGER, ForeignKey("tasks.id", ondelete="SET NULL"))
+    category_id = Column(
```

Comment:
I think better to add many to many support and allow users choose many labels per document

Reply:
But don't we have a tree-like structure of categories? So we could add labels like this? Or I miss something?

Reply:
I mean, user may choose categories from different subtrees and we might support this functionality